low-level/message_handlers/node_data_msg_handler.py

```diff
@@ -706,6 +709,10 @@ def _nw_cable_alert_exists(self, interfaces):
             interface_name = interface.get("ifId")
             phy_link_status = interface.get("nwCableConnStatus")
 
+            # If interface is eno2, skip processing it since it has very limited use right now

```

Comment:
LDR R1 : Skip 'eno2' network interface monitoring as its unused, so no alerts should come for same

Reply:
done